cmd/kpod/common.go

```diff
@@ -8,12 +8,15 @@ import (
 	"github.com/fatih/camelcase"
 	"github.com/kubernetes-incubator/cri-o/libkpod"
 	"github.com/urfave/cli"
+	"os"
```

Comment:
I think this file needs `gofmt` run on it - this should be above the rest of the imports

Reply:
There should be a empty line between golang imports and github.com imports, with the golang standard ones going first.

Reply:
fixed.